server/views/pages/calculation/checkInformation.njk

```diff
@@ -31,8 +31,12 @@
 {% block content %}
     {% set validationErrors = model.validationErrors %}
     {% include "../partials/customErrorBanner.njk" %}
+    {% include "../partials/ersedRecallNotificationBanner.njk" %}
```

Comment:
It looks like you’ve moved the banner on the view journey but not the calc journey? The calc summary page has a different view model to the sentence and offence page, so you’ll need to move some of the functions too.